assets/js/services/jwtApi.ts

```diff
@@ -0,0 +1,42 @@
+type AuthTokenKey = "authJwt";
+type RefreshTokenKey = "refreshJwt";
+
+const authTokenKey: AuthTokenKey = "authJwt";
+const refreshTokenKey: RefreshTokenKey = "refreshJwt";
+
+export type AuthResponseData = {
+  authToken: AuthTokenKey;
+  refreshToken: RefreshTokenKey;
+};
+
+export const setJwt = ({ authToken, refreshToken }: AuthResponseData) => {
+  authToken && localStorage.setItem(authTokenKey, authToken);
+  refreshToken && localStorage.setItem(refreshTokenKey, refreshToken);
```

Comment:
Shouldn't we add some alert/log when `authToken` or `refreshToken` is undefined? Because this is probably because of a bad return from the backend. Don't I think correctly?

Reply:
I've added an if for this. The error is not silenced anymore. 🤫❌